src/main/resources/js/view.commentTab.toolbar-init.js

```diff
@@ -0,0 +1,169 @@
+/*
```

Comment:
cool, I did not know we can use ES6 syntax. Could you post a link to atlassian's tutorial page you used for the code in this file?

Reply:
https://developer.atlassian.com/server/jira/platform/extending-the-rich-text-editor-in-jira/